base/searcher/searcher.Deployment.yaml

```diff
@@ -30,7 +30,7 @@ spec:
       - name: searcher
         env:
         - name: SEARCHER_CACHE_SIZE_MB
-          value: "100000"
+          value: "50000"
```

Comment:
Looks good. Maybe we should avoid duplication here, and instead use something like

```yaml
valueFrom:
  fieldRef:
    fieldPath: spec.resources.requests.ephemeral-storage
```

That will require us updating the cache parsing code to understand kubernetes resource limit specifications, but that would be a good thing. Not for this PR, but what do you think?